apis/sgv2-docsapi/start-docsapi.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+# Startup script for Stargate Document API
+#   This script can be used to start the Document API with an uber-jar.
+#   Any Java properties/options provided as arguments to this script are passed along to the `java` command.
```

Comment:
passing props does not work:

```
[13:04:53] ise:sgv2-docsapi <jeff/api-release-artifacts>$ ./start-docsapi.sh -Dstargate.data-store.ignore-bridge=true
   _____ __                         __            ____                       ___    ____  ____
  / ___// /_____ __________ _____ _/ /____       / __ \____  __________     /   |  / __ \/  _/
  \__ \/ __/ __ `/ ___/ __ `/ __ `/ __/ _ \     / / / / __ \/ ___/ ___/    / /| | / /_/ // /
 ___/ / /_/ /_/ / /  / /_/ / /_/ / /_/  __/    / /_/ / /_/ / /__(__  )    / ___ |/ _____/ /
/____/\__/\__,_/_/   \__, /\__,_/\__/\___/    /_____/\____/\___/____/    /_/  |_/_/   /___/
                    /____/

                                                               Powered by Quarkus 2.15.3.Final
INFO  [main] 2023-01-19 13:05:37,738 OpenApiRecorder.java:48 - CORS filtering is disabled and cross-origin resource sharing is allowed without restriction, which is not recommended in production. Please configure the CORS filter through 'quarkus.http.cors.*' properties. For more information, see Quarkus HTTP CORS documentation
WARN  [main] 2023-01-19 13:05:37,764 GrpcServerRecorder.java:108 - Using legacy gRPC support, with separate new HTTP server instance. Switch to single HTTP server instance usage with quarkus.grpc.server.use-separate-server=false property
INFO  [vert.x-eventloop-thread-1] 2023-01-19 13:05:37,897 GrpcServerRecorder.java:223 - gRPC Server started on 0.0.0.0:9000 [SSL enabled: false]
INFO  [main] 2023-01-19 13:05:37,930 Channels.java:216 - Creating Netty gRPC channel ...
INFO  [main] 2023-01-19 13:05:37,962 DataStorePropertiesConfiguration.java:59 - DataStoreConfig.ignoreBridge() == false, will try to fetch data store metadata using the Bridge
INFO  [main] 2023-01-19 13:05:38,083 DataStorePropertiesConfiguration.java:108 - Data store metadata fetch using Bridge failed (call #1/5), problem: (io.grpc.StatusRuntimeException) UNAVAILABLE: io exception
INFO  [main] 2023-01-19 13:05:40,916 DataStorePropertiesConfiguration.java:108 - Data store metadata fetch using Bridge failed (call #2/5), problem: (io.grpc.StatusRuntimeException) UNAVAILABLE: io exception
INFO  [main] 2023-01-19 13:05:44,000 DataStorePropertiesConfiguration.java:108 - Data store metadata fetch using Bridge failed (call #3/5), problem: (io.grpc.StatusRuntimeException) 
```

Reply:
fixing order. java props have to come before -jar option